### ISSUE TITLE
word.php displays an empty window.

### ISSUE THREAD

**Issue by chaoko99** [flan/hymmnoserver#8]
In the past, (Prior to migration I assume) it would pop out a window with some additional information on the word, as well as an easy way to pull up related words. Now it just displays a page with no body text like so.
![image](https://github.com/flan/hymmnoserver/assets/4706052/f8b3f706-649d-4ad7-be4f-a2ab015cc2d1)

```
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
"http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">

<?php/*
All code, unless otherwise indicated, is original, and subject to the terms of
the GNU GPLv3 or, at your option, any later version of the GPL.

All content is derived from public domain, promotional, or otherwise-compatible
sources and published uniformly under the
Creative Commons Attribution-Share Alike 3.0 license.

See license.README for details.
 
(C) Neil Tallim, 2009-2021
*/?>

<?php/*
All code, unless otherwise indicated, is original, and subject to the terms of
the GNU GPLv3 or, at your option, any later version of the GPL.

All content is derived from public domain, promotional, or otherwise-compatible
sources and published uniformly under the
Creative Commons Attribution-Share Alike 3.0 license.

See license.README for details.
 
(C) Neil Tallim, 2009
*/?>

<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <title>Hymmnoserver - Word</title>
        <link rel="icon" type="image/vnd.microsoft.icon" href="[./favicon.ico](view-source:https://hymmnoserver.uguu.ca/favicon.ico)"/>
<link rel="stylesheet" href="[./resources/hymmnoserver.css](view-source:https://hymmnoserver.uguu.ca/resources/hymmnoserver.css)"/>
<script type="text/javascript" src="[./resources/hymmnoserver.js](view-source:https://hymmnoserver.uguu.ca/resources/hymmnoserver.js)"></script>
<meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type"/>
    </head>
    <body style="width: 500px;">

```
    
    These are being called by the JS function `javascript:popUpWord('FOO', 50)` , but i don't really know enough about JS to summon any diagnosing information for you.

**Comment by flan** [comment]
Thanks for reporting this. 

Unfortunately, I won't be able to take at look at it in detail until later this week, but I don't think it's a JavaScript issue. More likely, as you suggested, it's related to the migration and some path I may have changed in the internal PHP mapping.

**Comment by flan** [comment]
This should now be corrected with https://github.com/flan/hymmnoserver/commit/35b845c23c646f880aef99329b3a495cfa757236, with changes being live on the site. It started when I did the last PHP update, which broke an implicit resolution of unqualified tokens to variables. (Which was always a bad thing, but PHP is a language steeped in legacy, so I'm not too surprised I overlooked it)

Thanks again for pointing it out!
